source/_docs/free-https.md

```diff
@@ -32,13 +48,9 @@ Check the current state of DNS propagation from different parts of the world usi
 
 ## Require HTTPS (Highly Recommended)
 
-### 301 Redirects
```

Comment:
let's keep this section as is, 301 redirects as a subhead of Require HTTPS, but below HSTS